nntrainer/models/neuralnet.cpp

```diff
@@ -375,34 +367,36 @@ sharedConstTensors NeuralNetwork::forwarding(
       node->forwarding(training);
     } else {
       /**
-       currently, it supports FSU asynch mode for inference. The prcedure of
-       FSU is below,
+         currently, it supports FSU asynch mode for inference. The prcedure of
+         FSU is below,
 
-       Prerequests : This function is called node by node at the forwarding
-       function in network graph.
+         Prerequests : This function is called node by node at the forwarding
+         function in network graph.
 
-       Step 1. If the execution order is the first (f==0) then, it will try to
-               load tensors which used at layer 0.
+         Step 1. If the execution order is the first (f==0) then, it will try
+       to load tensors which used at layer 0.
 
-       Step 2. It check whether these tensors from Step 1, then do the
-               forwarding of the first node.
+         Step 2. It check whether these tensors from Step 1, then do the
+                 forwarding of the first node.
 
-       Step 3. Then check the look a head which says how many layer weights need
-               to be loaded before running to hide overehad due to FSU,
+         Step 3. Then check the look a head which says how many layer weights
+       need to be loaded before running to hide overehad due to FSU,
 
-       Step 4. Try to get the tesors by asking tensors for layers which is done
-               by thread pool
+         Step 4. Try to get the tesors by asking tensors for layers which is
+       done by thread pool
```

Comment:
It seems like the indentation is incorrect.

Reply:
oh... thank you for review